nixos/modules/services/networking/gns3-server.nix

```
@@ -227,14 +233,27 @@ in {
         User = "gns3";
         WorkingDirectory = "%S/gns3";
 
+        # Required for ubridge integration to work
+        #
+        # GNS3 needs to run SUID binaries (ubridge)
+        # but NoNewPrivileges breaks execution of SUID binaries
+        DynamicUser = lib.mkForce false;
+        NoNewPrivileges = lib.mkForce false;
+        RestrictSUIDSGID = lib.mkForce false;
+        PrivateUsers = lib.mkForce false;
```

Why are these using `lib.mkForce`? If users have to change them to adapt their needs, do they have to use `lib.mkVmOverride`? Just `false` would be okay.

---

Thank you for your review! :)

We set this to false because GNS3 is compatible with these hardening flags.

I changed these flags to:

```suggestion
        DynamicUser = false;
        NoNewPrivileges = false;
        RestrictSUIDSGID = false;
        PrivateUsers = false;
```